content/docs/dvclive/api-reference/live/log_image.md

```diff
@@ -0,0 +1,77 @@
+# Live.log_image()
+
+Saves the given image `val` to the output file `name`.
+
+```py
+def log_image(name: str, val):
+```
+
+#### Usage:
+
+```py
+from dvclive import Live
+
+live = Live()
+
+import numpy as np
+img_numpy = np.ones((500, 500), np.uint8) * 255
+live.log_image("numpy.png", img_numpy)
+
+from PIL import Image
+img_pil = Image.new("RGB", (500, 500), (250, 250, 250))
+live.log_image("pil.png", img_pil)
+```
+
+## Description
+
+Supported values for `val` are:
+
+- A valid numpy array (convertible to image via
+  [PIL.Image.fromarray](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.fromarray))
+- A `PIL.Image` instance.
```

Comment:
From the [core PR](https://github.com/iterative/dvclive/pull/189/files#diff-8a38661b2789aafa7a10bea7661fbedca108942b12a84521c9745b2e6cb61d78R165-R169) it looks like paths to an image file are also accepted?

Reply:
Never mind I was confused with the `name` param... 🤔